config.go

```diff
@@ -18,7 +18,7 @@ type Configs struct {
 	AppPath                 string          `env:"app_path,required"`
 	ExpansionfilePath       string          `env:"expansionfile_path"`
 	Track                   string          `env:"track,required"`
-	UserFraction            string          `env:"user_fraction,opt[0.05,0.1,0.2,0.5]"`
+	UserFraction            string          `env:"user_fraction,range]0.0..1.0["`
```

Comment:
the type is string, AFAIK stepconf will not parse/validate it this way.

Reply:
Updated to float